src/custom/pages/Claim/ClaimsTable.tsx

```diff
@@ -116,15 +112,13 @@ const ClaimsTableRow = ({
   )
 }
 
-export default function ClaimsTable({ isAirdropOnly, hasClaims }: ClaimsTableProps) {
+export default function ClaimsTable({ isAirdropOnly, claims, hasClaims }: ClaimsTableProps) {
   const { selectedAll, selected, activeClaimAccount, claimStatus, isInvestFlowActive } = useClaimState()
 
   const { setSelectedAll, setSelected } = useClaimDispatchers()
 
   const pendingClaimsSet = useAllClaimingTransactionIndices()
 
-  const userClaimData = useUserEnhancedClaimData(activeClaimAccount)
-
```

Comment:
Passing down as props to avoid fetching the same thing in 3 different places and cause additional UI delays